app/controllers/index.js

```diff
@@ -1,23 +1,22 @@
 /**
  * Module dependencies.
  */
-var mongoose = require('mongoose'),
-    async = require('async'),
-    _ = require('underscore');
+var mongoose = require('mongoose'), //eslint-disable-line
+    async = require('async'), //eslint-disable-line
+    _ = require('underscore'); //eslint-disable-line
 
-/**
- * Redirect users to /#!/app (forcing Angular to reload the page)
- */
-exports.play = function(req, res) {
+
+// Redirect users to /#!/app (forcing Angular to reload the page)
+exports.play = function(req, res) { //eslint-disable-line
   if (Object.keys(req.query)[0] === 'custom') {
     res.redirect('/#!/app?custom');
   } else {
     res.redirect('/#!/app');
   }
 };
 
-exports.render = function(req, res) {
-    res.render('index', {
-        user: req.user ? JSON.stringify(req.user) : "null"
-    });
-};
\ No newline at end of file
+exports.render = function(req, res) { //eslint-disable-line
+  res.render('index', {
+    user: req.user ? JSON.stringify(req.user) : 'null'
```

Comment:
Why do we have `null` in quotes? I think It should be just `null` and not a string. @saheedt